ui/src/screens/Jobs/JobsScreen.tsx

```diff
@@ -50,6 +50,15 @@ const JobsScreen: React.FC = function () {
     ),
   );
 
+  const onSearchChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const body = {text : e.target.value};
+    JobsGoWhereApiClient.post<PostInterface[]>(`${process.env.REACT_APP_API}/jobs/search`, body).then(
```

Comment:
We can consider adding in a debounce to fire off the search sometime after the user has stopped typing in their query. Now it would just keep posting to the API whenever the input field is populated. 

https://www.freecodecamp.org/news/debounce-and-throttle-in-react-with-hooks/

Reply:
Good idea. Also, we already have the `throttle-debounce` [package](https://www.npmjs.com/package/throttle-debounce) in this project.

Reply:
Done. I couldn't use useCallback since the call requires an argument.